plugin/trino-openlineage/src/main/java/io/trino/plugin/openlineage/transport/OpenLineageHttpTransport.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.openlineage.transport;
```

Comment:
Since this is specific to `http` can we move to a package like `io.trino.plugin.openlineage.transport.http` and for Config object also so for kafka also we could have a dedicated package